File: BE/routine-service/src/main/java/com/hogumiwarts/lumos/routine/dto/RoutineResponse.java
package com.hogumiwarts.lumos.routine.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RoutineResponse {

	private Long routineId;
	private String routineName;
}

File: Android/data/src/main/java/com/hogumiwarts/data/entity/remote/Response/routine/RoutineResponse.kt
package com.hogumiwarts.data.entity.remote.Response.routine

data class RoutineResponse(
    val status: Long,
    val message: String,
    val data: List<RoutineData>
)

data class RoutineData(
    val routineId: Long,
    val routineName: String,
    val routineIcon: String,
    val gestureName: String
)